_posts/en/newsletters/2024-01-31-newsletter.md

```diff
@@ -0,0 +1,222 @@
+  [transaction sponsor][topic fee sponsorship] soft-fork. [...]
+  This requirement for all users to have a UTXO to pay fees negates the
+  efficiency of CTV-using UTXO sharing schemes [...] the only realistic
+  alternative is to use a third party to pay for the UTXO, eg via a LN
+  payment, but at that point it would be more efficient to pay an
+  [out-of-band mining fee][topic out-of-band fees]. That of course is
+  highly undesirable from a mining centralization perspective." (Links
+  added by Optech.) He recommends abandoning CTV and working instead on
+  [convenant schemes][topic covenants] that are compatible with
+  [RBF][topic rbf].
+
+    John Law replied that fee-dependent timelocks (see [Newsletter
+    #283][news283 fdt]) could make CTV safe to use with endogenous fees
+    in cases where particular versions of a transaction needed to be
+    confirmed by a deadline, although fee dependent timelocks might
+    delay some contract settlements by an indefinite amount of time.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 2.4.0-rc1][] is a release candidate for the next version of this
+  package providing a common interface to multiple different hardware
+  signing devices.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+- [Bitcoin Core #29291][] adds a test that will fail if a transaction
+  executing an `OP_CHECKSEQUENCEVERIFY` opcode appears to have a
+  negative version number.  This test, if it had been run by alternative
+  consensus implementations, would have caught the consensus failure bug
+  mentioned in [last week's newsletter][news286 bip68ver].
+
+- [Eclair #2811][], [#2813][eclair #2813], and [#2814][eclair #2814]
```

Comment:
@t-bast I'd love your review on this item to make sure I've drawn the correct inference!

Reply:
That's 100% correct, and is the right conclusion, privacy becomes really good even with a single trampoline node, which is great for mobile wallets!